.typo-ci.yml

```diff
@@ -0,0 +1,9 @@
+# Any typos we should ignore?
+excluded_words:
+  - Devnol
+  - Rexogamer
+  - akis
+  - akisblack
+  - Midou
+  - Midou360
+  - Midou36O
```

Comment:
Don't forget Miicord too

Reply:
Miicord is just a name for your Discord isn't it?

Reply:
it is but he said that and eh why not